backend/public-graph/graph/resolver.go

```diff
@@ -391,31 +391,29 @@ func (r *Resolver) GetOrCreateErrorGroup(ctx context.Context, errorObj *model.Er
 
 		environmentsString := getIncrementedEnvironmentCount(ctx, errorGroup, errorObj)
 
+		updatedState := errorGroup.State
+
+		if updatedState == privateModel.ErrorStateResolved {
+			updatedState = privateModel.ErrorStateOpen
+		}
```

Comment:
this is what toggles a resolved error getting a new instance to be open? could we add a comment to express that, as it's a bit confusing what this logic is for.

Reply:
Added in b4946ada087149827901336432060a6cc82a0b40

I still am unable to track down if this is a regression or if this ever actually worked but at least there's a test now to validate it.